### ISSUE TITLE
Does parallel provisioning work ? 

### ISSUE THREAD

**Issue by johntdyer** [mitchellh/vagrant-aws#63]
@mitchellh  So the readme for Vagrant 1.2.0 mentions Parallel provisioning, but I see no additional information on this.

```
Providers can now parallelize! If they explicitly support it, Vagrant will run "up" and other commands in parallel. For providers such AWS, this means that your instances will come up in parallel. VirtualBox does not support this mode.  
```

Does this work ? Do I need to enable it ? I ask because I have a multiVM AWS deploy which is happening serially right now.  Thanks!

**Comment by jayunit100** [comment]
`vagrant provision --parallel` gives me ...
```
Usage: vagrant provision [vm-name] [--provision-with x,y,z]
        --provision-with x,y,z       Enable only certain provisioners, by type.
    -h, --help                       Print this help
```

**Comment by jayunit100** [comment]
good news is `vagrant up` provisions in parallel by default in ec2, so the feature is there but its just that `--parallel` option is broken for provisioning.  see #273

**Comment by utdrmac** [comment]
@jayunit100 so was the option fixed? I see you closed #273 without specifying whether or not the problem was fixed.

**Comment by jayunit100** [comment]
Not fixed;  closed as dupe
